lib/elixir/test/elixir/kernel/warning_test.exs

```diff
@@ -9,379 +9,392 @@ defmodule Kernel.WarningTest do
   end
 
   test "unused variable" do
-    output = capture_err(fn ->
-      Code.eval_string """
-      defmodule Sample do
-        def hello(arg), do: nil
-
-        if true do
-          user = :warning
-        else
-          :nothing
+    output =
+      capture_err(fn ->
+        Code.eval_string("""
+        defmodule Sample do
+          def hello(arg), do: nil
+
+          if true do
+            user = :warning
+          else
+            :nothing
+          end
         end
-      end
-      """
-    end)
+        """)
+      end)
+
     assert output =~ "variable \"arg\" is unused"
     assert output =~ "variable \"user\" is unused"
   after
-    purge Sample
+    purge(Sample)
   end
 
   test "unused variable in redefined function in different file" do
-    output = capture_err(fn ->
-      Code.eval_string """
-      defmodule Sample do
-        defmacro __using__(_) do
-          quote location: :keep do
-            def function(arg)
+    output =
+      capture_err(fn ->
+        Code.eval_string("""
+        defmodule Sample do
+          defmacro __using__(_) do
+            quote location: :keep do
+              def function(arg)
+            end
           end
         end
-      end
-      """
-      Code.eval_string("""
-      defmodule RedefineSample do
-        use Sample
-        def function(var123), do: nil
-      end
-      """, [], file: "redefine_sample.ex")
-    end)
+        """)
+
+        Code.eval_string(
+          """
```

Comment:
Please move the code out to a variable.

Reply:
I don't get the comment can you explain again

Reply:
Did you mean ```eval = Code.eval_string(...)``` and then replace the  ```Code.eval_string``` with ```eval```

Reply:
He means to extract the string of code out to a variable so that the `Code.eval_quoted/3` call gets on one line only:

```elixir
code = """
...
"""

Code.eval_quoted(code, [], file: "redefine_sample.ex")
```

Reply:
Do i have to do only for the highlighted cases over here or all the possible code in that file?

Reply:
this what i did for the one of the test case

```
  test ":__struct__ is ignored when using structs" do
    assert capture_err(fn ->

            code ="""
               assert %Kernel.WarningTest.User{__struct__: Ignored, name: "joe"} ==
                      %Kernel.WarningTest.User{name: "joe"}
               """
             Code.eval_string(code, [], __ENV__)

           end) =~ "key :__struct__ is ignored when using structs"

```

Reply:
@m13m only for the cases where Code.eval_string is called with more than one argument. Also please remember to format the file one last time after your changes.